spyder/plugins/console/widgets/main_widget.py

```diff
@@ -323,13 +323,19 @@ def set_help(self, help_plugin):
         """
         self.shell.help = help_plugin
 
-    def handle_exception(self, text, is_traceback, is_pyls_error=False,
-                         is_faulthandler_report=False):
+    @Slot(dict)
+    def handle_exception(self, error_data):
         """
         Exception ocurred in the internal console.
 
         Show a QDialog or the internal console to warn the user.
         """
```

Comment:
Please complete this docstring to include what error_data is (using numpy docstrings notation)

Reply:
Fixed